pkg/test/framework/test.go

```diff
@@ -69,6 +70,12 @@ func (t *Test) Label(labels ...label.Instance) *Test {
 	return t
 }
 
+// Label applies the given labels to this test.
+func (t *Test) LabelFeatures(labels ...label.Instance) *Test {
```

Comment:
Maybe just `Feature`?

Reply:
I think there may be occasions where a single test covers multiple features or scenarios.  In particular, if we have two test cases that we decide to merge into one, we may need to keep both feature labels for reporting...

Reply:
My comment was really about getting rid of "Label" from the name. WRT singular vs plural, the other method is also vararg and is just called `Label` ... we should probably be consistent.